crates/bitwarden-crypto/src/xchacha20.rs

```diff
@@ -0,0 +1,137 @@
+//! # XChaCha20Poly1305 operations
+//!
+//! Contains low level XChaCha20Poly1305 operations used by the rest of the crate.
+//!
+//! In most cases you should use the [EncString][crate::EncString] with
+//! [KeyEncryptable][crate::KeyEncryptable] & [KeyDecryptable][crate::KeyDecryptable] instead.
+
+use chacha20poly1305::{AeadCore, AeadInPlace, KeyInit, XChaCha20Poly1305};
+use generic_array::{typenum::U24, GenericArray};
+
+/**
```

Comment:
Small nit, but I would move this comment to the top of the file and use `//!` like on the other one. As it is right now, we're technically adding a doc comment to `use crate::CryptoError`